libs/api/domains/health-insurance-v2/src/lib/graphql/models/accidentNotificationStatus.model.ts

```diff
@@ -0,0 +1,27 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+import { AccidentNotificationAttachment } from './accidentNotificationAttachment.model'
+import { AccidentNotificationConfirmation } from './accidentNotificationConfirmation.model'
+import { IsEnum } from 'class-validator'
+
+export enum StatusTypes {
+  ACCEPTED = 'Accepted',
+  REFUSED = 'Refused',
+  INPROGRESS = 'InProgress',
+  INPROGRESSWAITINGFORDOCUMENT = 'InProgressWaitingForAttachment',
+}
+
+@ObjectType()
+export class AccidentNotificationStatus {
```

Comment:
Should follow the [graphql naming convention](https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions#integrating-naming-conventions-into-shared-api).